crm-project/src/main/java/CRM/entity/DTO/AttributeDTO.java

```diff
@@ -0,0 +1,27 @@
+package CRM.entity.DTO;
+
+import CRM.entity.*;
+import lombok.*;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class AttributeDTO {
+
+    private Long id;
+    private Board belongsTo;
```

Comment:
If I understand correctly, according to your code, we send back the whole property of "board" to the user, when we return an attributeDTO. Right?
I don't think we need that. 
For starter - we never return a whole object and only a DTO one. Having said that, you need to change it to BoardDTO.
Furthermore, we don't even want to return the whole object. Just return it's id.

Reply:
replaced by id